Volume/Views/Onboarding/FollowView.swift

```diff
@@ -45,24 +43,29 @@ extension OnboardingView {
                         }
                     }
                 }
-                .background(Color.volume.backgroundGray)
+                .padding(.bottom, 48)
+                .background(
+                    GeometryReader { proxy in
+                        let offset = -proxy.frame(in: .named(scrollViewCoordinateSpace)).origin.y
+                        Color.volume.backgroundGray
+                            .preference(key: OffsetPreferenceKey.self, value: offset)
+                    }
+                )
+            }
+            .coordinateSpace(name: scrollViewCoordinateSpace)
+            .onPreferenceChange(OffsetPreferenceKey.self) {
```

Comment:
Could you explain what this is doing? Also, are you using TrackableScrollView anymore anywhere in the app? If not then can we delete the file?

Reply:
I actually made a GitHub mistake on this PR by starting this branch from the one I submitted in the onboarding scroll fix PR, so I'll address this issue in the that PR page.